src/modules/access-wallet/hardware/components/AccessWalletPaths.vue

```diff
@@ -9,9 +9,6 @@
         -->
         <div class="subtitle-1 font-weight-bold grey--text">STEP 2.</div>
         <div class="headline font-weight-bold">Select HD Derivation Path</div>
```

Comment:
can you add "mb-5"  class to Select HD Derivation Path div (otherwise there is no space once you removed bottom div